content/docs/getting-started/one-off-tasks.md

```diff
@@ -7,15 +7,13 @@ aliases:
 - /getting-started/cf-ssh/
 ---
 
-There are several options for running tasks in cloud.gov.
+This table shows a handful of common operations, with the recommended way to perform them in GovCloud.
```

Comment:
Agree! Also something went wrong with the table:

![screen shot 2017-04-24 at 4 28 41 pm](https://cloud.githubusercontent.com/assets/391313/25362824/2f6d89ee-290b-11e7-99c0-ea6ccfc881c9.png)